pkgs/tools/text/rgxg/default.nix

```diff
@@ -0,0 +1,19 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "rgxg";
+  version = "0.1.2";
+
+  # fetchFromGitHub does not work because of how its build ( and leaveDotGit is considered bad )
```

Comment:
LOL :). The situation is much more complicated then that so I think there's no need to explain. autoconf based tools are common in Nixpkgs and it's well known that they need an official "tarball" and can't easily be built with a bare git checkout, with or without the `.git` directory.

Reply:
In this sense I would prefer `fetchzip`. At least it doesn't garble the hash sum.
But it doesn't matter now, it is working.

Reply:
:-)

Reply:
Why did we switch to `fetchzip`?